src/tecmp2plp.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+	Copyright (c) 2020-2021 Technica Engineering GmbH
+	GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+*/
+
+#include <array>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <map>
+
+#include <tecmp/tecmp.h>
+#include <light_pcapng_ext.h>
+#include "pcapng_exporter/endianness.h"
+#include "pcapng_exporter/lin.h"
+#include "pcapng_exporter/linktype.h"
+#include "pcapng_exporter/pcapng_exporter.hpp"
+#include "pcap.h"
+#include <args.hxx>
+
+//AnAm: NOTE: Since the proper flag could not be found in the
+// TECMP documentation a temporary variable has been generated
+// to provide a temporary logic.
+#define TMP_ERROR_NODE_ACTIVE 0x0002
+#define NANOS_PER_SEC 1000000000
+
+#define PCAP_NG_MAGIC_NUMBER 0x0A0D0D0A
+#define PCAP_MAGIC_NUMBER 0xA1B2C3D4
+#define PCAP_MAGIC_NUMBER_LITTLE_ENDIAN 0xD4C3B2A1
+
+using namespace pcapng_exporter;
+
+#if _WIN32
+// From https://nmap.org/npcap/guide/npcap-tutorial.html
+#include <Windows.h>
+#include <tchar.h>
+bool LoadNpcapDlls()
+{
+	TCHAR npcap_dir[512];
+	UINT len;
+	len = GetSystemDirectory(npcap_dir, 480);
+	if (!len) {
+		fprintf(stderr, "Error in GetSystemDirectory: %x", GetLastError());
+		return false;
+	}
+	_tcscat_s(npcap_dir, 512, TEXT("\\Npcap"));
+	if (SetDllDirectory(npcap_dir) == 0) {
+		fprintf(stderr, "Error in SetDllDirectory: %x", GetLastError());
+		return false;
+	}
+	return true;
+}
+#endif
+
+void transform(
+	PcapngExporter exporter,
+	light_packet_interface packet_interface,
+	light_packet_header packet_header,
+	const uint8_t* packet_data
+) {
+	char *data;
+	memcpy(data,packet_data,packet_header.captured_length);
```

Comment:
Code format: space after commas
```suggestion
	memcpy(data, packet_data, packet_header.captured_length);
```